pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -202,6 +209,61 @@ func (ctrl *Controller) Run(workers int, stopCh <-chan struct{}) {
 
 	<-stopCh
 }
+func (ctrl *Controller) filterAPIServer(apiServer *configv1.APIServer) {
+	if apiServer.Name == "cluster" {
```

Comment:
reversing the logic here would remove the nested if.

```suggestion
	if apiServer.Name != "cluster" {
	  return
	}  
```